flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ArchivedExecutionGraph.java

```diff
@@ -100,22 +102,25 @@
 	@Nullable
 	private final String stateBackendName;
 
+	private Map<IntermediateDataSetID, PersistedIntermediateResultDescriptor> persistedIntermediateResult;
+
 	public ArchivedExecutionGraph(
-			JobID jobID,
-			String jobName,
-			Map<JobVertexID, ArchivedExecutionJobVertex> tasks,
-			List<ArchivedExecutionJobVertex> verticesInCreationOrder,
-			long[] stateTimestamps,
-			JobStatus state,
-			@Nullable ErrorInfo failureCause,
-			String jsonPlan,
-			StringifiedAccumulatorResult[] archivedUserAccumulators,
-			Map<String, SerializedValue<OptionalFailure<Object>>> serializedUserAccumulators,
-			ArchivedExecutionConfig executionConfig,
-			boolean isStoppable,
-			@Nullable CheckpointCoordinatorConfiguration jobCheckpointingConfiguration,
-			@Nullable CheckpointStatsSnapshot checkpointStatsSnapshot,
-			@Nullable String stateBackendName) {
+		JobID jobID,
+		String jobName,
+		Map<JobVertexID, ArchivedExecutionJobVertex> tasks,
+		List<ArchivedExecutionJobVertex> verticesInCreationOrder,
+		long[] stateTimestamps,
+		JobStatus state,
+		@Nullable ErrorInfo failureCause,
+		String jsonPlan,
+		StringifiedAccumulatorResult[] archivedUserAccumulators,
+		Map<String, SerializedValue<OptionalFailure<Object>>> serializedUserAccumulators,
+		ArchivedExecutionConfig executionConfig,
+		boolean isStoppable,
+		@Nullable CheckpointCoordinatorConfiguration jobCheckpointingConfiguration,
+		@Nullable CheckpointStatsSnapshot checkpointStatsSnapshot,
+		@Nullable String stateBackendName, Map<IntermediateDataSetID, PersistedIntermediateResultDescriptor>
```

Comment:
The argument should probably be put in a separate line with a `@Nullable` annotation as well.